contracts/vaults/Deposit.sol

```diff
@@ -7,8 +7,11 @@ contract Deposit is Base {
     using SafeERC20 for IERC20;
 
     function depositReward(uint256 _amount) external {
+        uint256 balanceBefore = swapToken.balanceOf(address(this));
         rewardToken.transferFrom(address(msg.sender), address(this), _amount);
-        emit DepositReward(_amount, address(rewardToken));
+        uint256 rewardTokenReceived = swapToken.balanceOf(address(this)) - balanceBefore;
```

Comment:
I like the balance checking, but you should use the same token

Reply:
oops! thanks